packages/@aws-cdk-containers/ecs-service-extensions/test/service.test.ts

```diff
@@ -25,87 +22,6 @@ describe('service', () => {
     }).toThrow(/Service 'my-service' must have a Container extension/);
 
 
-  });
-
-  test('should be able to add a container to the service', () => {
```

Comment:
Why is this test being removed? It seems unrelated to this change.

Reply:
I moved all `Container` extension related tests to a new test file `container.test.ts` (you can find this test there). I felt that as we add more properties to this extension, it would be better to have a separate test for it than to append it to a test file for another Construct.
Would you suggest I make it a new PR?